### ISSUE TITLE
State as Expanded is not Announced by Screen Reader on 'Select Location' Combo Box in 'Licenses and apps' Control.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14087]
### Issue: ###

State as expanded is not announced by screen reader on 'Select Location' combo box in 'Licenses and apps' control.

### Test Environment: ###

Test Environment:
OS Build: Version 21H2(22000.613)
Browser: Edge Anaheim Dev Version 92.0.891.1 (Official build) dev (64-bit)
Screen reader: Non-Visual Desktop Access (NVDA) (2021.1)

### Repro steps: ###

1. Open above URL: https://admin-ignite.microsoft.com/Adminportal/Home and login with valid credentials.
2. Navigate to Navigation Menu > Setting > Domain.
3. Navigate to table data and activate through enter key .
4. Navigate to 'User' tab by pressing right arrow key and activate through enter key.
5. Navigate to table data and activate through enter key .
6. Navigate to 'Licenses and apps' tab and activate through enter key.
7. Navigate to 'Select Location' combo box and activate through enter key.
8. Verify that state as 'expanded' is announced by screen reader or not.

### Actual result: ###

State as expanded is not announced by screen reader on 'Select Location' combo box in 'Licenses and apps' control.

### Expected result: ###

State as 'expanded' should be announced by screen reader on 'Select Location' combo box in 'Licenses and apps' control.

### Video: ###

https://user-images.githubusercontent.com/93735775/187530488-505e2179-37e7-4ab9-9815-52bfc64ed74e.mp4

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.
